content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -4107,13 +4107,46 @@ const addTag = await prisma.post.update({
 })
 ```
 
+### <inlinecode>unset</inlinecode>
+
+<Admonition type="warning">
+
+This method is available on MongoDB only in versions
+[3.11.1](https://github.com/prisma/prisma/releases/tag/3.11.1) and later.
+
+</Admonition>
+
+Use `unset` to unset the value of a scalar list. Unlike `set: null`, `unset` removes the list entirely.
```

Comment:
Is the focus on scalar lists here intended?

Reply:
(I think it is, I just have a hard time reviewing snippets while needing to expand the sections to figure out where I am)

Reply:
Yes, intended, it's part of the 'Scalar list methods' subsection